src/databricks/labs/ucx/configure.py

```diff
@@ -0,0 +1,68 @@
+import logging
+
+from databricks.sdk.service import compute
+
+logger = logging.getLogger(__name__)
+CLUSTER_ID_LENGTH = 20  # number of characters in a valid cluster_id
+
+
+class ConfigureClusterOverrides:
+    """Installation configuration operations to suplement install.WorkspaceInstaller"""
+
+    def __init__(self, ws, _choice_from_dict):
```

Comment:
```suggestion
    def __init__(self, ws, choice_from_dict):
```
underscores mean private vars